src/main/java/com/autotune/utils/KruizeConstants.java

```diff
@@ -517,6 +517,7 @@ public static final class KRUIZE_CONFIG_ENV_NAME {
         public static final String SETTINGS_HIBERNATE_TIME_ZONE = "hibernate_timezone";
         public static final String PLOTS = "plots";
         public static final String LOCAL = "local";
+        public static final String LOG_HTTP_REQ_RESP = "logAllHttpReqAndResp";
```

Comment:
String Name should match it should be 
logAllHttpReqAndResp

please test if true and false is working by setting it from env